docs/openapi-starter.md

```diff
@@ -0,0 +1,51 @@
+# How to implement openapi-starter
+
+## About the openapi-starter project
+
+The [openapi-starter](https://github.com/Redocly/openapi-starter) is a Redocly project that you can clone to your own GitHub account to more easily manage API definition files either as single files or multiple files.
+
+It automatically creates the required folder structure and generates a basic OpenAPI definition file, which is great if you're new to API reference docs.
+
+Although you can create your own folder structure, we recommend letting `openapi-starter` set it up initially (you can always make changes later). The folder structure is also key when using the `split` and `bundle` commands, which are used to split up large definitions for easier maintenance, and compile the consituent files back into a single file when you're ready to publish your API reference docs.
+
+## How to implement openapi-starter
+
+### Step 1: Copy the openapi-starter project
+
+1. Go to [https://github.com/Redocly/openapi-starter](https://github.com/Redocly/openapi-starter).
+2. Click the **Use this template** button.
+3. You’ll be asked to create a new repository from openapi-starter.
+4. Give your repo a meaningful name (e.g. redocly-openapi-starter).
+5. Choose whether you want it to be public or private.
+6. Leave the **Include all branches** option deselected.
+7. Click the **Create repository from template** button.
+
+The project is copied to your GitHub account. It is independent and not linked to the source.
+
+### Step 2: Clone the project
+
+This step clones the project to your local machine so you can use it with [OpenAPI CLI](.docs/quickstart.md).
+
+:::info
+We use [GitHub Desktop](https://desktop.github.com/) in this example, but go ahead and use whatever cloning method you like.
```

Comment:
Personally I'd like to avoid using screenshots as they just mean more maintenance. Could we compromise and only add information about the command line method?

Reply:
I've added an example for the CLI method alongside GitHub Desktop, without screenshots.